In [138]:
# import av lite pythonbibliotek som används till beräkningarna.
import numpy as np
import astropy.constants as acon
import scipy.constants as con

# Universums struktur och olika skalor

## Ljusår

Hur långt färdas ljuset på en sekund?

$c = 299792458 \frac{m}{s}$

Hur många varv runt jorden är det på en sekund?

Jordens radie är $R_j = 6371 km =6371000 m$

Jordens omkrets blir då $J_o = 2 \pi R_j$

In [139]:
# hämtar värdet på pi och sparar i variabeln pi
pi = np.pi
Rj = 6371e3
Jo = 2*pi*Rj
Jo

40030173.59204114

$J_o = 40030173.5 m$

Antal varv blir då $\frac{c}{J_o}$

In [140]:
c = 299792458
c/Jo

7.489162076968989

Antal varv = 7.48

Vad är då ett ljusår? Ett ljusår är så långt ljuset hunnit färdas på ett år.

På ett år går det 365.25 dagar. 1 dag är 24 timmar. 1 timme är 60 min. 1 min är 60 sek. Antal sekunder på ett år blir då

In [141]:
# ysek = antal sekunder på ett år
ysek = 365.25*24*60*60
ysek

31557600.0

Sekunder på ett år = 31557600 s

Avståndet ljuset färdas på ett år blir då $c \cdot y_{sek}$

In [142]:
c*ysek

9460730472580800.0

Ett ljusår defineras därför som _$1ly = 9460730472580800m$_ eller ungefär $9.46 \cdot 10^{12} km$

## Jorden till månen

Avståndet från jorden till månen är ca $D_m = 384000000 m = 384000 km$

<img src="earth_to_moon.png" alt="drawing" width="700"/>

### Övning
1) Hur lång tid tar det för ljuset att åka från jorden, till månen och tillbaka?

2) Hur många resor runt jorden motsvarar det att åka till månen?

3) Om jorden skulle ha samma diameter som en fotboll 22cm hur långt bort skulle månen befinna sig då? Om vi antar samma skalor.

### Lösning

1)

In [143]:
# Dm = avstånd till månen i meter
Dm = 384000e3
t = Dm/c
# vi tar två gånger t eftersom vi är intresserade av ersan till månen och tilbaka
2*t

2.5617722511218077

Det tar alltså ca 2.5 sekunder att skicka en signal fram och tillbaka från jorden till månen.

2)

In [144]:
Dm/(2*pi*Rj)

9.592763796466459

Att åka till månen innebär alltså att man skulle behöva åka ca 9.6 resor runt jorden

3)

In [145]:
# jordens diameter = 2*Rj
# vi får en skalfaktor som vi kallar faktor
faktor = Dm/(2*Rj)
faktor

30.13655627060116

In [146]:
# Bd = Bollens diameter (diametern på en fotboll är 22cm)
Bd = 22e-2

# multiplicera bollens diameter med faktorn
Bd*faktor

6.630042379532256

Månen skulle alltså befinna sig 6.6 meter från bollen

## Jorden till solen

$D_s = 149597870700 m = 1.4959787 \cdot 10^{11} m$

<img src="sun_earth.jpg" alt="drawing" width="700"/>

### Övning

1) Hur många resor runt jorden motsvarar det att åka till solen?

2) Om jorden hade samma diameter so en fotboll, hur långt bort skulle solen befinna sig då. Om vi antar samma skalor.

3) Hur stor diameter skulle solen ha då?

4) Hur många jordklot får man plats med innuti solen?


### Lösningar

1)

In [147]:
#Ds = avstånd till solen från jorden i meter
# Jo = Jordens omkrets som vi definerade i förra uppgiften
Ds = 1.4959787e11
Ds/Jo

3737.1276858450415

Det skulle alltså ta ca 3737 resor runt jorden för att komma fram till solen!

2)

In [148]:
# Jd = jordens diameter
Jd = 2*Rj
faktor = (Ds/Dj)
faktor

11740.532883377806

In [149]:
# Bd = bollens diameter
Bd*faktor

2582.9172343431173

Om jorden var stor som en fotboll skulle solen ligga ca 2.5km bort. Så om bollen är i klassrummet skulle solen ligga ungefär i Tullinge centrum.


3)

In [150]:
# Rs = Radien för solen
faktor = 2*Rs/(2*Rj)
Bd*faktor

24.023544184586406

Om jordens diameter är stor som en fotboll, dvs 22cm, så skulle solen i Tullinge centrum vara stor som en sfär med 24 meters diameter

4) Volymen för ett klot är $V = \frac{4 \pi r^3}{3}$

In [151]:
# Jv = Jordens volym
Jv = 4*pi*(Rj**3)/3

# Rs = solens radie
Rs = 6.95700e8 

# Sv = Solens volym
Sv = 4*pi*(Rs**3)/3
Sv/Jv

1302096.5698420107

Det får alltså plats 1.3 miljoner jordklot innuti solens volym.

### Astronomisk enhet (AU)

För att lättare kunna hantera avstånden i solsystemet använder vi avståndet mellan solen och jorden som ett standardmått.

Vi definerar därför en astronomisk enhet som $1AU = 149597870700 m = 1.4959787 \cdot 10^{11} m$

## Solsystemet

I den här bilden är skalorna inte korrekt. Det är mycket större avstånd mellan alla objekt än hur det ser ut i bilden. Men värdena på axlarna ger korrekta avstånd numeriskt.

<img src="solar_system.png" alt="drawing" width="1200"/>

Hur många Astronomiska enheter går det på ett ljusår?

In [152]:
# en astronomisk enhet AU är avståndet till solen
AU = Ds
AU

149597870000.0

In [153]:
# ly = avståndet ljusår i meter
ly = c*ysek
ly

9460730472580800.0

In [154]:
ly/AU

63241.07738018462

Det går 63241 AU på ett ly

## Oorts kometmoln

<img src="oort-cloud.jpg" alt="drawing" width="1200"/>

## Interstellära grannskapet

<img src="solar_neighborhood.png" alt="drawing" width="900"/>

Alfa centauri ligger 4.37 ly från solen och är vår närmaste grannstjärna

## Vintergatan (Milky way)

<img src="milky_way.png" alt="drawing" width="1000"/>

## Lokala galaxghopen


<img src="local_group.png" alt="drawing" width="900"/>

Avståndet mellan vintergatan och andromedagalaxen är ca 2.5 Mly

## Hubble Deep field

Hubbleteleskopet zoomade in på den lila vita rutan bredvid texten "XDF" och tog en bild med lång exponeringstid, dvs den samlade in allt ljus från området under en längre tid. Resultatet kan ni se på bilden nedan.


<img src="hubble_deep_moon.jpg" alt="drawing" width="900"/>

De flesta ljuspunkter i den här bilden är en galax. Ljuset som vi ser är det samlade ljuset från alla de miljardtals stjärnor som finns i varje galax.


<img src="hubble_deep.jpg" alt="drawing" width="900"/>

## laniakea superkluster


<img src="laniakea_supercluster.png" alt="drawing" width="1000"/>

## lokala superclustren


<img src="local_superclusters.jpg" alt="drawing" width="1000"/>

## Observerbara universum


<img src="observable_universe.jpg" alt="drawing" width="1000"/>